Cargo.toml

```diff
@@ -53,6 +53,8 @@ getrandom = { version = "=0.2.15", features = ["js"] }
 hex = { version = "0.4", features = ["serde"] }
 iai = "0.1"
 itertools = "0.12.1"
+jemalloc-ctl = { version = "0.5" }
+jemallocator = { version = "0.5" }
```

Comment:
As mentioned [here](https://github.com/o1-labs/proof-systems/pull/3079/files#r2089285483), I think it would be better to use the maintained fork: 
```toml
[...]
[dependencies]
[...]
tikv-jemallocator = "0.5"
```

Reply:
Even version 0.6 (latest release)

Reply:
Actually I had to go for 0.5 because I cannot find `stats` in `tikv-jemalloc-ctl:0.6`...